cmd/core-data/res/configuration.toml

```diff
@@ -1,6 +1,6 @@
 [Writable]
 DeviceUpdateLastConnected = false
-MetaDataCheck = false
+MetaDataCheck = true
```

Comment:
I think this change is not related with this PR

Reply:
Correct. This is part of what appears to be a cherry-picked commit from my feature branch above. It is intended to demonstrate how the correlation ID is passed from core-data to core-metadata, but should not necessarily be part of the default implementation.